lib/resty/session.lua

```diff
@@ -354,6 +405,7 @@ function session.start(opts)
         return opts, opts.present
     end
     local self, present = session.open(opts)
+    touch(self)
```

Comment:
Should we only do this, if the `save` is not called below in this function?

Reply:
yes, that's a better aproach